### ISSUE TITLE
[Accessibility]A11y_VSCode_ActivityBar_Keyboard: Setting option of "ActivityBar" is access by the mouse only

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52014]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : MAC OS HighSierra10.13.5

**Additional Details:**
Keyboard,MAS2.1.1,MAS2.4.3

**Repro Steps:**
1.Navigate to "Activity Bar" at left side of the screen after login.
3.Navigate through keyboard such as "Tab" key.
4.Observed that, "Setting" option is not accessible by the keyboard keys.

**Actual:**
"Setting" option of "ActivityBar" is not selected by the keyboard. user have to use mouse to access the "Setting" option. 

**Expected:**
Without using mouse "Setting" option should be accessible.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**User Impact:**
Keyboard using user will not able to access the "Setting" option.

**MAS Reference:**
MAS2.1.1- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)
MAS2.4.3- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_ActivityBar_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2105686/Accessibility.A11y_VSCode_ActivityBar_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by sandy081** [comment]
@AccessibilityTestingTeam-TCS I can able to access it using keyboard. 

Please let me know if I am missing something here.

**Comment by AccessibilityTestingTeam-TCS** [comment]
The issue is mentioned in attached ppt. This issue occured in MAC, when user is on activity bar and move to next item of activity bar the using tab user can navigate upto "Extensions" button only. Here keyboard trap is occured and user is not able to move to "Setting" option.Here without keyboard trapp all the control should be accessible by the keys.
